recipes/openvdb/all/conanfile.py

```diff
@@ -245,3 +295,41 @@ def package_info(self):
         self.cpp_info.components["openvdb-core"].names["cmake_find_package"] = "openvdb"
         self.cpp_info.components["openvdb-core"].names["cmake_find_package_multi"] = "openvdb"
         self.cpp_info.components["openvdb-core"].set_property("cmake_target_name", "OpenVDB::openvdb")
+
+
+        #NanoVDB
+        if Version(self.version) >= "9.0.0" and self.options.nanovdb:
+            self.cpp_info.components["nanovdb"].bindirs = []
+            self.cpp_info.components["nanovdb"].libdirs = []
+            self.cpp_info.components["nanovdb"].requires = []
+            if self.options.nanovdb_with_openvdb:
+                self.cpp_info.components["nanovdb"].requires.append("openvdb-core")
+            if self.options.nanovdb_with_tbb:
+                self.cpp_info.components["nanovdb"].requires.append("onetbb::onetbb")
+            if self.options.with_zlib:
+                self.cpp_info.components["nanovdb"].requires.append("zlib::zlib")
+            if self.options.with_blosc:
+                self.cpp_info.components["nanovdb"].requires.append("c-blosc::c-blosc")
+
+            if self.settings.os in ("Linux", "FreeBSD"):
+                self.cpp_info.components["nanovdb"].system_libs = ["pthread"]        
+
+            self.cpp_info.components["nanovdb"].defines.append("NANOVDB_USE_INTRINSICS")
+
+            if self.settings.os == "Windows":
+                self.cpp_info.components["nanovdb"].defines.append("_USE_MATH_DEFINES")
+                self.cpp_info.components["nanovdb"].defines.append("NOMINMAX")
+                self.cpp_info.components["nanovdb"].defines.append("TBB_USE_PREVIEW_BINARY")
+
+            if self.options.nanovdb_with_openvdb:
+                self.cpp_info.components["nanovdb"].defines.append("NANOVDB_USE_OPENVDB")
+            if self.options.nanovdb_with_tbb:
+                self.cpp_info.components["nanovdb"].defines.append("NANOVDB_USE_TBB")
+            if self.options.with_zlib:
+                self.cpp_info.components["nanovdb"].defines.append("NANOVDB_USE_ZIP")
+            if self.options.with_blosc:
+                self.cpp_info.components["nanovdb"].defines.append("NANOVDB_USE_BLOSC")
+
+            self.cpp_info.components["nanovdb"].names["cmake_find_package"] = "nanovdb"
```

Comment:
For new components and new things in general, it is fine (and recommended) to drop support for ``cmake_find_package[_multi]`` generators

Reply:
Done